SwiftUIBasics/Views/PlansView.swift

```diff
@@ -8,11 +8,108 @@
 import SwiftUI
 
 struct PlansView: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+       var body: some View {
+        VStack(alignment:.center){
+            Text("Choose")
+                .font(.system(size: 30, design: .serif))
+                .fontDesign(.rounded)
+                .fontWeight(.black)
+            Text("Your Plan")
+                .font(.system(size: 30, design: .serif))
+                .fontDesign(.rounded)
+                .fontWeight(.black)
+            HStack {
+                CardView(iconName: "", title: "Basic", description: "Per month", price: "$9")
```

Comment:
```suggestion
                CardView(title: "Basic", description: "Per month", price: "$9")
```
IconName could have a default value